src/encoded/static/components/dataset.js

```diff
@@ -2201,6 +2201,67 @@ const functionalCharacterizationSeriesTableColumns = {
     },
 };
 
+
+/**
+ * Generate a comma-separated string of pulse-chase time durations from the given experiment's biosamples.
+ * @param {object} experiment - The experiment to generate the pulse-chase durations from.
+ * @returns {string} - A comma-separated string of pulse-chase durations.
+ */
+const computePulseDurations = (dataset) => {
+    const biosamples = collectDatasetBiosamples(dataset);
+    let pulseDurations = '';
+    if (biosamples.length > 0) {
+        biosamples.forEach((biosample) => {
+            if (biosample.pulse_chase_time) {
+                pulseDurations = `${biosample.pulse_chase_time} ${biosample.pulse_chase_time_units}${biosample.pulse_chase_time > 1 ? 's' : ''}`;
+            }
+        });
+    }
+    pulseDurations = _.uniq(pulseDurations);
+    return pulseDurations;
+};
+
+const pulseChaseTimeSeriesTableColumns = {
+    accession: {
+        title: 'Accession',
+        display: (experiment, meta) => (
+            <span>
+                {meta.adminUser || publicDataset(experiment) ?
+                    <a href={experiment['@id']} title={`View page for experiment ${experiment.accession}`}>{experiment.accession}</a>
+                :
+                    <span>{experiment.accession}</span>
+                }
+            </span>
+        ),
+    },
+
+    pulseDuration: {
+        title: 'Pulse-chase duration',
+        getValue: (experiment) => computePulseDurations(experiment),
+    },
+
+    biosample_summary: {
+        title: 'Biosample summary',
+    },
+
+    lab: {
+        title: 'Lab',
+        getValue: (experiment) => (experiment.lab ? experiment.lab.title : null),
```

Comment:
Fancy! I adjusted to your suggested syntax, and kept the line- it might be that the check is needed in this context, since many other SeriesColumns definitions in this file have that line